Makefile

```diff
@@ -9,6 +9,7 @@ GOVET_IGNORE_RULES = \
 # List of executables needed for 'make generate'
 GENERATE_DEPENDENCIES = \
 	github.com/golang/mock/mockgen \
+	github.com/gogo/protobuf/protoc-gen-gogoslick \
```

Comment:
Commented on PR - note if they say no, then we should abandon protobuf/grpc, lol :) Oh man.